### PR TITLE
Fix the parameters of a test fixture

### PR DESCRIPTION
**Pull request by Zay11Zay**
### Pull Request Checklist

Signed-off-by: Isaiah Singletary <zaysingletary@gmail.com>

### CODE REVIEW

File: changelog.d/1.bugfix [hunk 1]
```diff
@@ -0,0 +1 @@
+The parameters in the [test_events.py](synapse/tests/rest/client/v1/test_events.py) in the prepare method were in the wrong order. This change fixes the order. Contributed by Isaiah Singletary.
```
Comment (clokep) [inline changelog.d/1.bugfix:1]: Pleae rename this file to `7243.misc` (7243 to match the PR number and "misc" because it is a test only change.)

Comment (clokep) [reply changelog.d/1.bugfix:1]: Also, I think we can simplify it to say "Correct the parameters of a test fixture. Contributed by Isaiah Singletary."

**Comment by clokep** [comment]
Related to #7242

File: changelog.d/1.bugfix (hunk 1 above)
Comment (clokep) [reply changelog.d/1.bugfix:1]: Looks good! Please delete this file and we can merge this.

**Comment by richvdh** [comment]
please could this PR have a name which reflects what it fixes?

**Pull request merged**